tracer/build/_build/UpdateVendors/VendoredDependency.cs

```diff
@@ -72,6 +72,14 @@ static VendoredDependency()
                 pathToSrc: new[] { "sketches-dotnet-1.0.0", "src", "Datadog.Sketches" },
                 // Perform standard CS file transform with additional '#nullable enable' directive at the beginning of the files, since the vendored project was built with <Nullable>enable</Nullable>
                 transform: filePath => RewriteCsFileWithStandardTransform(filePath, originalNamespace: "Datadog.Sketches", AddNullableDirectiveTransform));
+            Add(
+                libraryName: "IndieSystem.Text.RegularExpressions",
+                version: "0.1",
+                downloadUrl: "https://github.com/robertpi/IndieRegex/archive/refs/tags/v0.3.zip",
+                pathToSrc: new[] { "IndieRegex-0.3", "src" },
+                // Perform standard CS file transform with additional '#nullable enable' directive at the beginning of the files, since the vendored project was built with <Nullable>enable</Nullable>
+                transform: filePath => RewriteCsFileWithStandardTransform(filePath, originalNamespace: "IndieSystem.Text.RegularExpressions",
+                    Add31OrGreader, AddNullableDirectiveTransform, AddIgnoreNullabilityWarningDisablePragma, FixupResources, RemoveHashCode));
```

Comment:
typo + we have built-in functionality to exclude files:
```suggestion
                    AddIfNetcoreapp31OrGreater, AddNullableDirectiveTransform, AddIgnoreNullabilityWarningDisablePragma, FixupResources),
                relativePathsToExclude: new[] { "additional/HashCode.cs" });

);
```